python/ray/llm/_internal/serve/configs/server_models.py

```diff
@@ -108,6 +108,10 @@ def validate_dynamic_lora_loading_path(cls, value: Optional[str]):
 
 
 class ModelLoadingConfig(BaseModelExtended):
+    model_config = ConfigDict(
```

Comment:
@eicherseiji could you take a pass at other configs and make sure we have such safeguard globally where similar issue might happen?